spec/error_mapper_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+RSpec.describe JsonapiErrorsHandler::ErrorMapper do
+  let(:subject) { described_class }
+
+  before(:each) {
+    subject.class_variable_set(:@@mapped_errors, {}) 
+  }
+
+  describe '.mapped_errors' do
```

Comment:
In this `describe` block I understand a subject as calling a method: `described_class.mapped_errors` so it seems like having one subject at the very top of the file is too general. Also, if it's only a `described_class`, maybe better is just get rid of it?

Reply:
yes, agree :) let me update this one.